examples/mongodb/1.8/start-mongodb.sh

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
+
+set -e
+# enable job control used fg 
+set -m
+
+HOSTNAME=`hostname`
+
+OPTS=`getopt -o h: --long hostname: -n 'parse-options' -- "$@"`
+if [ $? != 0 ] ; then echo "Failed parsing options." >&2 ; exit 1 ; fi
+
+echo "$OPTS"
+eval set -- "$OPTS"
+
+while true; do
+  case "$1" in
+    -h | --hostname )     HOSTNAME=$2;        shift; shift ;;
+    -- ) shift; break ;;
+    * ) break ;;
+  esac
+done
+echo "Using HOSTNAME='$HOSTNAME'"
+
+function wait_mongodb_upandready {
+  # Wait until Mongo is ready to accept connections, exit if this does not happen within 30 seconds
+  COUNTER=0
+  until mongosh --quiet --eval "use admin; printjson(db.serverStatus());"
+  do
+    sleep 1
+    COUNTER=$((COUNTER+1))
+    if [ "${COUNTER}" -gt "30" ]; then
+      echo "MongoDB did not initialize within 30 seconds, exiting"
+      exit 2
+    fi
+    echo "Waiting for MongoDB to initialize... ${COUNTER}/30"
+  done
+}
+
+function deploy_replica_set {
+  wait_mongodb_upandready
+  mongosh -u admin -p admin --authenticationDatabase admin localhost:27017/inventory <<-EOF
+  rs.initiate( {
+     _id : "rs0",
+     members: [
+        { _id: 0, host: "${HOSTNAME}:27017" }
```

Comment:
@ismailsimsek on multi-node clusters/replica sets this section should look different. Can you look if you can generate this section if multiple cluster nodes are provided by an optional ENV var?

Reply:
@rk3rn3r sorry for the delay, now it should be possible to configure multiple comma separated hosts with `REPLICA_SET_HOSTS` variable. defaults `localhost`